pkg/templates/kubeadm/v1beta3/kubeadm.go

```diff
@@ -177,17 +178,29 @@ func NewConfig(s *state.State, host kubeoneapi.HostConfig) ([]runtime.Object, er
 		APIServer: kubeadmv1beta3.APIServer{
 			ControlPlaneComponent: kubeadmv1beta3.ControlPlaneComponent{
 				ExtraArgs: map[string]string{
-					"endpoint-reconciler-type": "lease",
-					"service-node-port-range":  cluster.ClusterNetwork.NodePortRange,
-					"enable-admission-plugins": kubeflags.DefaultAdmissionControllers(kubeSemVer),
+					"enable-admission-plugins":      kubeflags.DefaultAdmissionControllers(kubeSemVer),
+					"endpoint-reconciler-type":      "lease",
+					"kubelet-certificate-authority": "/etc/kubernetes/pki/ca.crt",
+					"profiling":                     "false",
+					"request-timeout":               "300s",
```

Comment:
Why is this 300s when the [upstream default](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-apiserver/) is 1m?

> --request-timeout duration     Default: 1m0s
>
> An optional field indicating the duration a handler must keep a request open before timing it out. This is the default request timeout for requests but may be overridden by flags such as --min-request-timeout for specific types of requests.

Reply:
Report says:
```
[WARN] 1.2.22 Ensure that the --request-timeout argument is set as appropriate (Manual)

1.2.22 Edit the API server pod specification file /etc/kubernetes/manifests/kube-apiserver.yaml
and set the below parameter as appropriate and if needed.
For example, --request-timeout=300s
```

Reply:
My understanding of that report is that they provided an example how to set the flag. I think we should use same values as used by default by API server.

Reply:
done